controllers/chat/group/changeChannelDetail.js

```diff
@@ -4,6 +4,7 @@ const ErrorResponse = require('../../../utils/response/ErrorResponse');
 const {ResponseSuccess} = require('../../../utils/Responses');
 const UsersFunction = require('../../../databases/functions/users');
 const {User} = require('../../../databases/models');
+const BetterSocialCore = require('../../../services/bettersocial');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `BetterSocialCore` is new in this hunk. Ensure that the path to the module is correct and that the module exports the required functions, especially `fcmToken.sendGroupChatNotification` which is used later in the code.
```diff
+ const BetterSocialCore = require('../../../services/bettersocial');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->